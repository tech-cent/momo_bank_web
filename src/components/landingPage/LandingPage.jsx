import React from 'react';
import { Container } from 'react-bootstrap';
import './LandingPage.scss';
import icon from '../../assets/images/google-play-store.svg';
import themeImage from '../../assets/images/MTN-Mobile-Money-outlet.jpeg';
import NavigationBar from '../navigationBar/NavigationBar';
import Footer from '../footer/Footer';

const LandingPage = () => {
  return (
    <>
      <NavigationBar/>
      <Container>
        <div className="row  body-container">
          <div className="col-md-5 landing-page-details">
            <h1>MTN momo Bank</h1>
            <p className="paragraph-text">
              The mobile money - bank integration solution <br />
              Make transfers from mobile money to your bank account as often as you wish.
              Once the money is transfered, you are able to make investments to earn profits<br/><br/>
              Take up investment opportunities such as fixed deposit accounts with great interest rates <br/><br/><br/>
            </p>
            <h5>Join us today</h5>
            <a href="https://play.google.com" rel="noopener noreferrer" target="_blank">
              <img className="google-play-store-icon" src={icon} alt="Link to Android App" scale="0" />
            </a>
          </div>
          <div className="col-md-5 landing-page-details">
            <img className="momo-bank-theme-image" src={themeImage} alt="Momo Bank" scale="0" />
          </div>
        </div>
      </Container>
      <Footer/>
    </>
  );
}

export default LandingPage;
