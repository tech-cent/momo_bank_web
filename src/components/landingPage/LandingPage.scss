.body-container {
  margin: 9% 0 0 0;
  padding: 3rem 2rem;
  font-family: averta,avenir w02,avenir,Helvetica,Arial,sans-serif;

  .landing-page-details {
    text-align: left;

    .google-play-store-icon {
      width: 135px;
      height: 40px;
    }
  }
  
}
