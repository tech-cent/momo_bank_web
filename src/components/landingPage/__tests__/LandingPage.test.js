import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../LandingPage';

describe('LandingPage component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<LandingPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
