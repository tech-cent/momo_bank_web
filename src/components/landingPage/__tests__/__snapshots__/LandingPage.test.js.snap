// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component should render correctly 1`] = `ShallowWrapper {}`;
